shell/platform/darwin/ios/framework/Source/FlutterUndoManagerPlugin.mm

```diff
@@ -48,46 +42,43 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
   }
 }
 
-- (NSUndoManager*)undoManager {
-  return _viewController.undoManager;
-}
-
-- (void)resetUndoManager API_AVAILABLE(ios(9.0)) {
```

Comment:
Removed `API_AVAILABLE(ios(9.0))` since the min is iOS 12.  Must have missed that in a cleanup.